src/main/java/com/google/sps/servlets/getDailyQuiz.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.QueryResultList;
+import java.text.DateFormat;
+import java.util.Date;
+import com.google.appengine.api.datastore.Key;
+
+@WebServlet("/getQuizStatus")
+public class getDailyQuiz extends HttpServlet {
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        Query query = new Query("Quiz");
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery pq = datastore.prepare(query);
+
+        Object time = null;
+        for(Entity entity : pq.asIterable()) {
+            time = entity.getProperty("timestamp");
+        }
```

Comment:
r u trying to get the time the quiz should release here? can u sort the datastore objects by time and get 1 value instead of iterating through all the times?

Reply:
Actually, the idea is that there will be only one Quiz entity in the datastore so I am looping through this just to grab a specific property of the Quiz entity which here is just the time the quiz was made

Reply:
oooh ok so maybe do the thing i mentioned above and just get the one entity instead of the loop.